flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+	"only-arches": ["x86_64"]
```

Comment:
that's disappointing given that "cross-platform" is in it's description.

Reply:
Cross-platform as in it supports Windows, macOS, and GNU+Linux. I don't think Tauri has ARM support.

Reply:
it does support. build from source.

Reply:
Please see [my other comment](https://github.com/flathub/flathub/pull/5709#discussion_r1798579209) asking how to build from source when using Deno which there is no dependency manifest generator for.